akka-remote/src/main/scala/akka/remote/serialization/ArteryMessageSerializer.scala

```diff
@@ -66,6 +72,8 @@ private[akka] final class ArteryMessageSerializer(val system: ExtendedActorSyste
     case ActorRefCompressionAdvertisementAck(from, id)      ⇒ serializeCompressionTableAdvertisementAck(from, id)
     case adv: ClassManifestCompressionAdvertisement         ⇒ serializeCompressionAdvertisement(adv)(identity)
     case ClassManifestCompressionAdvertisementAck(from, id) ⇒ serializeCompressionTableAdvertisementAck(from, id)
+    case RemoteWatcher.ArteryHeartbeat                      ⇒ serializeArteryHeartbeat()
```

Comment:
this can return `Array.empty` (and move the toByteArray to each line, unfortunately)

but put the `Array.empty[Byte]` in a `val` to avoid allocations